static/sass/styles.scss

```diff
@@ -284,3 +284,10 @@ summary {
     }
   }
 }
+
+// Advantage section
+#enterprise-table { // sass-lint:disable-line no-ids
+  .p-table tr:target {
```

Comment:
Won't a `<tr>` always be in a table pattern, overqualified selector?

Reply:
Good point